### ISSUE TITLE
completions with `--force` does not work

### ISSUE THREAD

**Issue by chriskrycho** [volta-cli/volta#656]
I wanted to update my completions with the tweaks we've made lately, and found that the `-f`/`--force` flag Is not working as expected:

```
volta completions fish -o ~/.config/fish/completions/volta.fish --force
error: Completions file `/Users/ckrycho/.config/fish/completions/volta.fish` already exists.

Please remove the file or pass `-f` or `--force` to override.
Error details written to /Users/ckrycho/.volta/log/volta-error-2020-02-04_10_39_22.456.log
```

**Comment by charlespierce** [comment]
Looking at the code, it looks like it should work. However, I also note that the error if the file exists [here](https://github.com/volta-cli/volta/blob/master/src/command/completions.rs#L43) is the same as the error if creating the file fails [here](https://github.com/volta-cli/volta/blob/master/src/command/completions.rs#L64). I wonder if this is a case of the latter failing for some unrelated reason (in-use or something by the shell, perhaps?) and then outputting the same error?

**Comment by charlespierce** [comment]
Also testing locally by creating the file and then trying to overwrite it, I see it correctly fail without the `--force` flag and succeed with `--force`

**Comment by chriskrycho** [comment]
I think this may have been a case of “hadn’t updated that machine” yet.  I’ll double-check tomorrow.

**Comment by chriskrycho** [comment]
Circling back to this: yes, hilariously, this was me reporting a bug *I myself had already fixed*; I just hadn't updated the version on my own machine. 🤦‍♂

**Comment by chriskrycho** [comment]
For anyone curious later: this was resolved by #647.
